.github/workflows/release.yaml

```diff
@@ -0,0 +1,125 @@
+    if: ${{ needs.test-release-needed.outputs.make-release }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Generate YAML
+        uses: ./generate-yaml
+        with:
+          excluded-dirs: .github
+
+      # Put release toolkit to work.
+      - name: Link dependencies
+        uses: ./link-dependencies
+      - name: Calculate next version
+        id: version
+        uses: ./next-version
+
+      # Prepare to commit the Changelog.
+      - name: Configure Git
+        run: |
+          git config user.name '${{ env.BOT_NAME }}'
+          git config user.email '${{ env.BOT_EMAIL }}'
+
+      # Create changelog and commit it.
+      - name: Update the markdown
+        uses: ./update-markdown
+        with:
+          version: ${{ steps.version.outputs.next-version }}
+      - name: Commit updated changelog
+        run: |
+          git add CHANGELOG.md
+          git commit -m "Update changelog with changes from ${{ steps.version.outputs.next-version }}"
+          git push -u origin ${{ github.event.repository.default_branch }}
+
+      # Create the release from the commit above using only the changelog for this release.
+      - name: Render the changelog snippet
+        uses: ./render
+        with:
+          version: ${{ steps.version.outputs.next-version }}
+      - name: Create release
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
+          gh release create \
+            ${{ steps.version.outputs.next-version }} \
+            --title ${{ steps.version.outputs.next-version }} \
+            --target $(git rev-parse HEAD) \
+            --notes-file CHANGELOG.partial.md
+
+      - name: Move major tag to this new commit
+        run: |
+          # The release is created above, the repo still does not have the tag available here
+          git fetch
+          git pull --tags
+
+          # Delete, tag, and push the major tag
+          git push --delete origin "${{ steps.version.outputs.next-version-major }}" || true  # Do not fail if the tag does not exist. It will be created.
+          git tag --force "${{ steps.version.outputs.next-version-major }}" "${{ steps.version.outputs.next-version }}"  # The tag might be deleted upstream bit still exists locally
+          git push origin "${{ steps.version.outputs.next-version-major }}"  # Push only the new tag
+
+          # Delete, tag, and push the major.minor tag
+          git push --delete origin "${{ steps.version.outputs.next-version-major-minor }}" || true  # Do not fail if the tag does not exist. It will be created.
+          git tag --force "${{ steps.version.outputs.next-version-major-minor }}" "${{ steps.version.outputs.next-version }}"  # The tag might be deleted upstream bit still exists locally
+          git push origin "${{ steps.version.outputs.next-version-major-minor }}"  # Push only the new tag
+
+  notify:
+    name: Test if release is needed
```

Comment:
Sorry I've just found this Copy&paste leftover

Reply:
Done at [#eb7ad2e](https://github.com/newrelic/release-toolkit/pull/187/commits/eb7ad2e551c9baee94ca3138d62f46183f0686ca)